GUIDE.md

```diff
@@ -1,50 +1,50 @@
 ## Interactor
-Содержит бизнес логику модуля. Нежелательно заставлять Interator ходить в сеть, обращаться к базе данных и т.д. Он выступает агрегатором бизнес логики. Так, например, для получения данных он обращается к Provider, к различным Worker'ам для выполнения каких-либо действий с данными, после чего передает подготовленные данные дальше в Presenter и т.д.
-Interactor получает запрос на операцию над данными от ViewController с помощью Request и предает данные в Presenter с помощью Response.
+Contains the business logic of the module. Interactor should not make network requests, access the database, etc. It acts as an aggregator of business logic. So, for example, to get data, it calls the Provider, to various Workers to perform any data actions, and then sends the prepared data further to the Presenter, etc.
+Interactor receives a request for an operation on data from the ViewController with the Request and submits the data to the Presenter using Response.
 
 ## Worker
-Реализует переиспользуемую часть бизнес логики (например: алгоритм фильтрация данных).
+Implements the reused part of the business logic (for example: algorithm for data filtering).
 
 ## Presenter
-Содержит логику подготовки данных для отображения (например: форматирование данных (номера телефона, суммы), локализация и т.д.) и передает их во ViewController посредством ViewModel.
+Contains the logic for preparing data for display (for example: data formatting (phone number, sum), localization, etc.) and passes them to ViewController via ViewModel.
 
 ## ViewController
-Отвечает за отображение состояния, полученого от Presenter и взаимодействует с пользователем. ViewController содержит в себе только одну view, которая создается в методе loadView(). Он не управляет расположением view. Вся необходимая верстка уже должна быть в отдельно реализованной view. Однако ViewController выступает делегатом для всего view слоя. Так он знает, когда изменился текст в TextField или пользователь нажал на ячейку.
+Responsible for displaying the status received from Presenter and interacting with the user. ViewController contains only one view, which is created in the loadView () method. It does not control the location of the view. All necessary layout should be already presented in a separately implemented view. However, ViewController acts as a delegate for the entire view layer. So it knows when the Text in TextField was changed or the user clicked on the cell.
 
 ```
-override func loadView() {
-    view = CatalogListView(delegate: self)
+override func loadView () {
+    view = CatalogListView (delegate: self)
 }
 ```
 
 ## Provider
-Абстракция отвечающая за доступ к данным. Инкапсулирует в себе обращение к Service для получения данных из сети (или других источников) и/или DataStore, если данные должны кэшироваться.
+Abstraction responsible for accessing data. Encapsulates a call to Service to retrieve data from the network (or other sources) and/or DataStore if the data should be cached.
```

Comment:
Provider is an abstraction responsible for accessing data. It encapsulates calls to Service to retrieve data from the network (or other sources) and/or DataStore if the data should be cached.